lighthouse-cli/printer.js

```diff
@@ -113,7 +113,8 @@ function createOutput(results, outputMode) {
   // Pretty printed.
   const bold = '\x1b[1m';
   const reset = '\x1B[0m';
-  let output = `\n\n${bold}Lighthouse results:${reset} ${results.url}\n\n`;
+  const version = results.meta.lighthouseVersion;
+  let output = `\n\n${bold}Lighthouse (${version}) results:${reset} ${results.url}\n\n`;
```

Comment:
Thought about a seperate line, but I figured it might be confusing. `Lighthouse (x.x.x)` is clear that it's LH version x.x.x that was ran. `Version: ${version}` might make it seem like that is the report/output version.